precompile/precompiletest/test_precompile.go

```diff
@@ -91,6 +95,10 @@ func (test PrecompileTest) setup(t testing.TB, module modules.Module, state *tes
 	t.Helper()
 	contractAddress := module.Address
 
+	if test.Rules == nil {
+		test.Rules = extras.AvalancheRules{IsGranite: true}
```

Comment:
why we set this here? If this is an absolute requirement we should provide the correct `AvalancheRules` like everything is activated up to `IsGranite` (i.e `IsFortuna, IsDurango: true`). 

I just put up a quick [PR](https://github.com/ava-labs/coreth/pull/1327), you can use the helper from `extrastest` to get the rules with the fork.